src/BuildScriptGenerator/Php/PhpBashBuildSnippet.sh.tpl

```diff
@@ -0,0 +1,13 @@
+﻿echo "PHP executable: $php"
+
+cd "$SOURCE_DIR"
```

Comment:
nit: this is no longer needed, as the base script makes sure every snippet runs in this folder